website/_docs/tutorial/query_pushdown.md

```diff
@@ -14,8 +14,10 @@ If a query can not be answered by any cube, Kylin supports pushing down such que
 ### Query Pushdown config
 
 #### Pushdown to single engine
-1. In Kylin's installation directory, uncomment configuration item `kylin.query.pushdown.runner-class-name` of config file `kylin.properties`, and set it to `org.apache.kylin.query.adhoc.PushDownRunnerJdbcImpl`
+Take hive as an example to explain the opening steps:
 
+1. In Kylin's installation directory, uncomment configuration item `kylin.query.pushdown.runner-class-name` of config file `kylin.properties`, and set it to `org.apache.kylin.query.adhoc.PushDownRunnerJdbcImpl`,
+If you need to pushdown the query to presto, please set this configuration to `org.apache.kylin . query.adhoc.PushDownRunnerJdbcImpl`. For other configurations, please refer to `push down to Presto` at the end of this page.
```

Comment:
Please correct me if I was wrong, here is what I can found:
- Is `PushDownRunnerJdbcImpl` the right class name?